apps/docs/components/Navigation/NavigationMenu/NavigationMenuRefList.tsx

```diff
@@ -28,6 +28,12 @@ const NavigationMenuRefList = ({
   }
 
   const filteredSections = commonSections.filter((section) => {
+    if (section.type === 'category') {
+      section.items = section.items.filter((item) => {
+        return !item.excludes?.includes(id)
+      })
+    }
+
```

Comment:
This allows items inside a section to be removed from the side nav when `exclude` is used.